packages/sit-onyx/src/styles/mixins/input.scss

```diff
@@ -21,6 +21,27 @@
     // optional marker should be displayed at the very end of the label
     &.onyx-optional-marker {
       justify-content: space-between;
+
+      &::after {
+        display: none;
+      }
+    }
+  }
+
+  &__optional {
```

Comment:
Should we create maybe a mixin so we don't have to copy the styles from "styles/required.scss"?